src/agent/agent_info/src/agent_info_persistance.cpp

```diff
@@ -157,11 +235,17 @@ void AgentInfoPersistance::SetUUID(const std::string& uuid)
     SetAgentInfoValue("uuid", uuid);
 }
 
+void AgentInfoPersistance::SetGroups(const std::vector<std::string>& group_list)
+{
+    SetAgentGroups(group_list);
```

Comment:
Why don't just set the groups in this function instead of calling another one?

Reply:
Done!